srcalc/src/test/java/gov/va/med/srcalc/web/view/VariableEntryTest.java

```diff
@@ -75,28 +102,52 @@ public final void testRetrievedCardiacAge()
     @Test
     public final void testWithRetrievedLabs()
```

Comment:
I really like this test now: it tests quite a bit of functionality with a good economy of code.